jenkins-ci/Jenkinsfile

```diff
@@ -1,164 +1,96 @@
+            }
+        }
+        stage('all-in-one: verify services'){
+            steps{
+                openshiftVerifyService apiURL: '', authToken: '', namespace: '', svcName: 'jaeger-query', verbose: 'false'
+                openshiftVerifyService apiURL: '', authToken: '', namespace: '', svcName: 'jaeger-collector', verbose: 'false'
+            }
+        }
+        stage('all-in-one: functional test'){
+            steps{
+                withEnv(["JAVA_HOME=${ tool 'jdk8' }", "PATH+MAVEN=${tool 'maven-3.5.2'}/bin:${env.JAVA_HOME}/bin"]) {
+                    sh '''
+                        export USE_COLLECTOR_OR_AGENT=agent
+                        rm ./jaeger-java-test -rf
+                        git clone https://github.com/Hawkular-QE/jaeger-java-test.git
+                        cd jaeger-java-test
+                        mvn clean test
+                    '''
+                }
+            }
+        }
+        stage('all-in-one: delete all') {
+            steps {
+                sh 'oc delete all,template,daemonset,configmap -l jaeger-infra'
+            }
+        }
+        stage('production: deploy Cassandra') {
+            steps {
+                sh 'oc create -f production/cassandra.yml'
+            }
+        }
+        /* TODO if possible verify Cassandra deployment before proceeding  */
+        stage('production: deploy Jaeger') {
+            steps {
+                sh 'oc process -f production/jaeger-production-template.yml | oc create -f -'
+            }
+        }
+        stage('production: verify Jaeger services'){
+            steps{
+                openshiftVerifyService apiURL: '', authToken: '', namespace: '', svcName: 'jaeger-collector', verbose: 'false'
+                openshiftVerifyService apiURL: '', authToken: '', namespace: '', svcName: 'jaeger-query', verbose: 'false'
+            }
+        }
+        stage('production: functional test'){
+            steps{
+                withEnv(["JAVA_HOME=${ tool 'jdk8' }", "PATH+MAVEN=${tool 'maven-3.5.2'}/bin:${env.JAVA_HOME}/bin"]) {
+                sh '''
+                    export USE_COLLECTOR_OR_AGENT=collector
+                    rm ./jaeger-java-test -rf
+                    git clone https://github.com/Hawkular-QE/jaeger-java-test.git
+                    cd jaeger-java-test
+                    mvn clean test
+                '''
+                }
+            }
+        }
+        stage('production: delete all') {
+            steps {
+                sh 'oc delete all,template,daemonset,configmap -l jaeger-infra'
+            }
         }
-      }
-    }
-    stage('production: delete all') {
-      steps {
-        sh '''
-          echo "Before clean environment:"
-          oc get all
-          oc delete all,template,daemonset,configmap -l jaeger-infra
-          echo "After clean environment:"
-          oc get all
-        '''
-      }
     }
-  }
+}
+
+  /* FIXME restore this
```

Comment:
@kevinearls restore this script, to show the detailed status of the build to the community.